tests/cephfs/cephfs_bugs/validate_mds_metadata.py

```diff
@@ -0,0 +1,166 @@
+from tests.cephfs.cephfs_utilsV1 import FsUtils
+from utility.log import Log
+from utility.retry import retry
+
+log = Log(__name__)
+
+
+def run(ceph_cluster, **kw):
+    try:
+        """
+         CEPH-83583721 - Validate the details of all MDS servers are displayed with command `ceph mds metadata`.
+
+        Steps:
+        1. Create Filesystem with 2 active and 1 standby
+        2. validate mds metadata info
+        3. Make 1 active and 2 standby and enable standby reply
+        4. validate mds metadata info
+        5. Stop active MDS and Validate metadata info
+        6. Stop Standby replay MDs and Validate metadata info
+
+        Cleanup:
+        1. unmount the FS
+        2. Remove FS
+
+        """
+        tc = "CEPH-83583721"
+        log.info("Running cephfs %s test case" % (tc))
+        fs_util = FsUtils(ceph_cluster)
+        config = kw.get("config")
+        clients = ceph_cluster.get_ceph_objects("client")
+        build = config.get("build", config.get("rhbuild"))
+        fs_util.prepare_clients(clients, build)
+        fs_util.auth_list(clients)
+        client1 = clients[0]
+        fs_name = "cephfs_mds_metadata"
+        fs_util.create_fs(client1, fs_name)
+        mds_nodes = ceph_cluster.get_nodes("mds")
+        host_list = [node.hostname for node in mds_nodes]
+        hosts = " ".join(host_list)
+        client1.exec_command(
+            sudo=True,
+            cmd=f"ceph orch apply mds {fs_name} --placement='3 {hosts}'",
+            check_ec=False,
+        )
+        client1.exec_command(sudo=True, cmd=f"ceph fs set {fs_name} max_mds 2")
+        fuse_mount_dir = "/mnt/fuse_" + "".join(
+            secrets.choice(string.ascii_uppercase + string.digits) for i in range(5)
+        )
+        fs_util.fuse_mount(
+            [client1],
+            fuse_mount_dir,
+            new_client_hostname="admin",
+            extra_params=f" --client_fs {fs_name}",
+        )
+        client1.exec_command(
+            sudo=True,
+            cmd=f"cd {fuse_mount_dir};"
+            f"mkdir subdir;"
+            f"dd if=/dev/urandom of=subdir/sixmegs bs=1M conv=fdatasync count=6 seek=0;",
+        )
+        log.info("Scenario 1: Validate Active MDS and check metadata")
+        validate_metadata_fs_status(
+            client1,
+            fs_name,
+        )
+
+        log.info("set allow_standby_replay to true ")
+        client1.exec_command(sudo=True, cmd=f"ceph fs set {fs_name} max_mds 1")
+        log.info("Scenario 2: Validate standby replay MDS and check metadata")
+        client1.exec_command(
+            sudo=True, cmd=f"ceph fs set {fs_name} allow_standby_replay 1"
+        )
+        fs_util.wait_for_standby_replay_mds(client1, fs_name)
+        log.info("Scenario 2: Validate standby replay MDS and check metadata")
+        validate_metadata_fs_status(client1, fs_name)
+
+        log.info("Scenario 3: Fail Active MDS and check metadata")
+        fs_util.get_active_mdss(client1, fs_name=fs_name)
```

Comment:
Do we need this function call get_active_mdss here? As we are executing same in the line 86 as well to capture the results.